### DISCUSSION TITLE
GitHub Actions Cache

### DISCUSSION THREAD

**Discussion by smoench** [ubicloud/ubicloud#1742] [Q&A] [answered]
Hey there 👋 ,

since https://github.com/ubicloud/ubicloud/pull/1668 and https://github.com/ubicloud/ubicloud/pull/1731 have been merged, I'm interested if it "save" to use the ubicloud github actions cache or is there anything we need to consider? I'm asking as there is no blog post neither anything in the docs. Are there any storage limits or will be something charged additionally?

Thanks in advance :slightly_smiling_face:

**Comment by byucesoy** [comment, marked as answer]
Hi @smoench :wave:,

We recently updated our docs. You can find more information regarding Github Actions Cache from [here](https://www.ubicloud.com/docs/github-actions-integration/ubicloud-cache).

**Reply by smoench** [reply]
Awesome, thank you!
